### PR TITLE
Handle 'g' group tag for NMEA TagBlock.

### PR DESCRIPTION
**Pull request by Leila-Codes**
Added support for decoding of NMEAv4 'g' tag block field.

Please let me know if you would like additional tests or any changes?

### CODE REVIEW

File: pyais/messages.py [hunk 1]
```diff
@@ -155,6 +155,51 @@ def wrapper(tb: 'TagBlock') -> typing.Any:
...
+    """Tag Block Group represents the 3-int group sequence 
+    optionally included as part of the NMEA Tag Block
+
+    it consists of 3, comma-seperated integers X-Y-Z where:
+    - X = Message ID in sequence
+    - Y = Total parts in group
+    - Z = Unique GroupID for this group of messages."""
+
+    __slots__ = (
+        'msg_id',
```
Comment (M0r13n) [inline pyais/messages.py:168]: I would prefer to name this attribute `sentence_number` or `sentence_num`

**Comment by M0r13n** [comment]
@Leila-Codes Hey there,

isn't the actual grouping of sentences missing in your PR?

File: pyais/messages.py (hunk 1 above)
Comment (Leila-Codes) [reply pyais/messages.py:168]: Of course. I'm very amenable on var names. Please comment any other suggestions you may have as well and I'll update them all in batch! 😇

**Comment by Leila-Codes** [comment]
Hmm, you're right that whilst these would enable decoding this doesn't actually perform any grouping by itself.

Are you suggesting changes to the decoder/iterators to use these tags as well?

Sorry I didn't look at that during this PR as my particular use-case required something more bespoke to handle grouping across parallel instances of the app.

File: pyais/messages.py (hunk 1 above)
Comment (M0r13n) [reply pyais/messages.py:168]: Let's name them `sentence_num` and `sentence_tot`

**Review by M0r13n — APPROVED** [review]

**Pull request merged**